config/routes.rb

```diff
@@ -5,23 +5,24 @@
   end
   devise_for :users, skip: [:registrations], controllers: { sessions: "sessions" }
 
-  resources :users, only: [:edit, :update]
-  resources :profiles
+  resources :users
 
   resources :organisations do
-    resources :memberships, except: [:index, :edit, :update, :show]
+    resources :memberships
   end
 
-  resources :permissions, only: [:new, :index, :create, :destroy]
-
   root "welcome#index"
 
   namespace :api, format: "json" do
     namespace :v1 do
+      get "me" => "users#me"
```

Comment:
Both work atm - `/api/v1/me` and `/api/v1/users/me`

I prefer the shorter version as putting `/users/me` doesn't lend any extra information